runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -678,10 +683,10 @@ public void handleClick(MenuOptionClicked event)
 		}
 
 		if ((event.getWidgetId() == WidgetInfo.BANK_ITEM_CONTAINER.getId()
-				|| event.getWidgetId() == WidgetInfo.BANK_INVENTORY_ITEMS_CONTAINER.getId())
+			|| event.getWidgetId() == WidgetInfo.BANK_INVENTORY_ITEMS_CONTAINER.getId())
```

Comment:
Revert this tab change. It is used to show nested boolean operations.
```suggestion
				|| event.getWidgetId() == WidgetInfo.BANK_INVENTORY_ITEMS_CONTAINER.getId())
```